lib/core/Head.js

```diff
@@ -150,10 +150,19 @@ class Head extends React.Component {
           })}
 
         {/* Site defined code. Keep these at the end to avoid overriding. */}
-        <link
-          rel="stylesheet"
-          href={this.props.config.baseUrl + 'css/main.css'}
-        />
+        {this.props.hash ? (
```

Comment:
Seems like we can save few lines of code by removing double re-use of <link ....

Example:
```js
href = {`${this.props.config.baseUrl}css/main${this.props.hash ? this.props.hash : ''}.css`}
```

or ( i prefer this )
```js
href = {this.props.config.baseUrl + 'css/main.' + this.props.hash ? this.props.hash : '' + '.css'}
```

Reply:
woah, this not crossed my mind before @endiliey. i think your second option is what we should do